### ISSUE TITLE
update broke the program

### ISSUE THREAD

**Issue by NoUserNameForYou** [TencentARC/GFPGAN#41]
I'm getting: 

Traceback (most recent call last):
  File "E:\location to GFPGAN\inference_gfpgan.py", line 98, in <module>
    main()
  File "E:\location to GFPGAN\inference_gfpgan.py", line 52, in main
    restorer = GFPGANer(
  File "E:\location to GFPGAN\gfpgan\utils.py", line 50, in __init__
    self.face_helper = FaceRestoreHelper(
TypeError: __init__() got an unexpected keyword argument 'device'

without BASICSR_JIt=TRUE

and with BASICR_JIT=TRUE it just hangs, resources are not used at all and it just waits and waits. Waited 30 minutes already. 

Old version works though.

Is it a BasicSR issue?

Also, the command in upscale_factor is only "upscale" now.

**Comment by xinntao** [comment]
1. It is because of the version issue. Please update the `basicsr` and `facexlib` to the latest version. 

2. 
> Also, the command in upscale_factor is only "upscale" now.

Thanks for reporting, it has been fixed.

**Comment by NoUserNameForYou** [comment]
Thank you friend but simply updating didn't work. I had to uninstall then re-install local copies with git clone comment. 

I'm on mingw64 windows 10 and what I did was:

git clone "basicsr's github.git" 
cd BasirSR
BASICSR_EXT=TRUE python setup.py develop  (it created the basicsr with cuda, I saw bunch of nvidia locations)
then cd .. back into gfpgan folder
git clone "facexlib's github.git"
cd facexlib
python setup.py develop
cd ..
python setup.py develop (and this builds gfpgan's latest version)

And now it works fine with cuda. Thanks for your help.
